readme.md

```diff
@@ -1,7 +1,7 @@
 # Boilerplate for layout tasks
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_snake/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_snake/report/html_report/)
+- [DEMO LINK](https://Aliud13.github.io/layout_snake/)
+- [TEST REPORT LINK](https://Aliud13.github.io/layout_snake/report/html_report/)
```

Comment:
Links doesn't work, did you do deploy?

Reply:
Yes, I think, I will make changes and I will deploy again